bahmni-docker/.env

```diff
@@ -0,0 +1,83 @@
+# Docker Compose Variables
+COMPOSE_PROFILES=default
+
+# Host-Port Mappings, credentials for Atomfeed Sync across various services. Defaults to services running in docker.
+OPENMRS_HOST=openmrs
+OPENMRS_PORT=8080
+OPENMRS_ATOMFEED_USER=admin
+OPENMRS_ATOMFEED_PASSWORD=test
+OPENELIS_HOST=openelis
+OPENELIS_PORT=8052
+OPENELIS_ATOMFEED_USER=atomfeed
+OPENELIS_ATOMFEED_PASSWORD=AdminadMIN*
+ODOO_HOST=odoo
+ODOO_PORT=8069
+ODOO_ATOMFEED_USER=admin
+ODOO_ATOMFEED_PASSWORD=admin
+
+# Shared Variables
+BAHMNI_CONFIG_PATH=
+
+# OpenElis Environment Variables
+OPENELIS_IMAGE_TAG=latest
+OPENELIS_DB_IMAGE_TAG=demo-latest
+OPENELIS_DB_SERVER=openelisdb
+OPENELIS_DB_DUMP_PATH=./openelis-db-dump
+
+# Odoo Environment Variables
+ODOO_IMAGE_TAG=latest
+ODOO_DB_IMAGE_TAG=demo-latest
+ODOO_DB_USER=odoo
+ODOO_DB_PASSWORD=odoo
+ODOO_DB_DUMP_PATH=./odoo-db-dump
+EXTRA_ADDONS_PATH=./extra-odoo-addons
+BAHMNI_ODOO_MODULES_PATH=
+
+#Odoo Connect Environment Variables
+ODOO_CONNECT_IMAGE_TAG=latest
+
+#OpenMRS Environment Variables
+OPENMRS_IMAGE_TAG=latest
+OPENMRS_DB_IMAGE_TAG=demo-latest
+OPENMRS_DB_NAME=openmrs
+OPENMRS_DB_HOST=openmrsdb
+OPENMRS_DB_USERNAME=openmrs-user
+OPENMRS_DB_PASSWORD=password
+OPENMRS_DB_CREATE_TABLES='false'
+OPENMRS_DB_AUTO_UPDATE='true'
+OPENMRS_MODULE_WEB_ADMIN='false'
+OPENMRS_DEBUG='true'
+OPENMRS_UPLOAD_FILES_PATH=./openmrs-uploads
+BAHMNI_OPENMRS_MODULES_PATH=
+MYSQL_ROOT_PASSWORD=adminAdmin!123
+
+#Bahmni Web Environment Variables
+BAHMNI_WEB_IMAGE_TAG=latest
+BAHMNI_APPS_PATH=
+
+#Bahmni Proxy Environment Variables
+PROXY_IMAGE_TAG=latest
+
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=welcome
+RABBITMQ_USERNAME=guest
+RABBITMQ_PASSWORD=guest
+GATEWAY_JWK_URL="https://dev.abdm.gov.in/gateway/v0.5/certs"
+GATEWAY_BASE_URL="https://dev.abdm.gov.in/gateway/v0.5"
+HIU_CLIENT_ID=
+HIU_CLIENT_SECRET=
+CONSENT_MANAGEMENT_URL="https://dev.ndhm.gov.in/cm"
+CONSENT_MANAGEMENT_SUFFIX="@sbx"
+CONSENT_NOTIFICATION_URL="http://localhost:8003/consent/notification"
+DATA_PUSH_URL="http://ec2-65-0-184-72.ap-south-1.compute.amazonaws.com/hiu-api/data/notification"
+IDENTITY_JWK_URL= https://dev.ndhm.gov.in/auth/realms/consent-manager/protocol/openid-connect/certs
+HIU_ID="Bahmni"
+HIU_NAME="Bahmni"
+HIP_SECRET=
+HIP_CLIENTID=
+GATEWAY_URL="https://dev.abdm.gov.in/gateway"
+OPENMRS_URL="http://openmrs:8080/openmrs"
+OPENMRS_PASSWORD="Admin123"
+CORS_ALLOWED_ORIGIN="http://ec2-65-0-184-72.ap-south-1.compute.amazonaws.com"
```

Comment:
We can remove the env to be checked in, we can place the value in while running the image

Reply:
removed